src/entitysdk/models/core.py

```diff
@@ -12,16 +15,36 @@ class Struct(BaseModel):
 class Identifiable(BaseModel):
     """Identifiable is a model with an id."""
 
-    id: int | None = None
-    update_date: datetime | None = None
-    creation_date: datetime | None = None
+    id: int | None = Field(
+        default=None,
+        examples=[1, 2, 3],
+        description="The primary key identifier of the resource.",
+    )
+    update_date: datetime | None = Field(
+        default=None,
+        examples=[datetime(2025, 1, 1)],
+        description="The date and time the resource was last updated.",
+    )
+    creation_date: datetime | None = Field(
+        default=None,
+        examples=[datetime(2025, 1, 1)],
+        description="The date and time the resource was created.",
+    )
```

Comment:
Unrelated to this PR, instead of `*_date` I'd prefer a different name on both the server and client side, because they are not dates (my vote goes to `created_at` and `updated_at`).
But let's keep them as they are for consistency with the backend.

Reply:
I am fine with the proposal. I'll change them once the server side is changed.